src/cgroups/v2/freezer.rs

```diff
@@ -0,0 +1,206 @@
+use anyhow::{Result, *};
```

Comment:
I feel the following code is more clear since this file only uses  `Result` and `bail!`.
```suggestion
use anyhow::{Result, bail};
```